htdocs/install/upgrade2.php

```diff
@@ -64,7 +64,7 @@
 // Ne fonctionne que si on est pas en safe_mode.
 $err=error_reporting();
 error_reporting(0);
-@set_time_limit(300);
+//@set_time_limit(300);
```

Comment:
If we remove this we got timeout on system where time out is set by default to 60 or 120 seconds.
Why removing this ? May be, you want to increase it to a very high value instead ?

Reply:
Yes because 300s is not enough to update version 10 to 11 (in my case)
Or increase it to 1200s for example or let the user to manage it in the ini of its php installation without remaining to look why it stops at 300s even by increasing the values in php and apache ... and will drop

Reply:
Why not create hidden conf so if you want to modify, you change a constant
`if (! empty($conf->global->MAIN_OVERRIDE_TIME_LIMIT)) etc`

Reply:
Yes we can!

Reply:
> 
> 
> Why not create hidden conf so if you want to modify, you change a constant
> `if (! empty($conf->global->MAIN_OVERRIDE_TIME_LIMIT)) etc`

done